examples/werewolf_game/start_game.py

```diff
@@ -1,16 +1,20 @@
 import asyncio
 import platform
 import fire
+import sys
```

Comment:
Try to avoid such code as much as possible

Reply:
ok. got it.  
it's just local testing code. i have delete it.